tests/spec/aggregate_frame/test_count_for_patient.py

```diff
@@ -1,44 +1,30 @@
-from databuilder.query_language import (
-    Dataset,
-    IdSeries,
-    build_event_table,
-    build_patient_table,
-)
-
-from ..helpers import transpose
-
-p = build_patient_table(
-    "p",
-    {
-        "patient_id": IdSeries,
-    },
-)
-
-
-e = build_event_table(
-    "e",
-    {
-        "patient_id": IdSeries,
-    },
-)
-
-
-def test_count_for_patient(in_memory_engine):
-    in_memory_engine.setup(
+from ..tables import e, p
+
+table_data = {
+    p: """
+          | i1
+        --+----
+        1 | 101
+        2 | 201
+        3 | 301
+        """,
```

Comment:
Do we need `p` here when it's not used directly? I think there is code in `spec_test()` to cope with it missing for population purposes. (Also applies to other tests below.)

Reply:
Oh, I've figured this out. You need it to show the result for members of the population missing from the event table. I think that deserves a comment.

Reply:
Good point, have added a comment.